editor/styling/stylesheets/editor.sass

```diff
@@ -48,6 +48,7 @@
 @import 'components/_text-area'
 @import 'components/_text-field'
 @import 'components/_titled-pane'
+@import 'components/_tooltip'
```

Comment:
Huh. So we had this before, but never imported it? 🤔

Reply:
Hehe, yes.